modules/waf/README.md

```diff
@@ -0,0 +1,38 @@
+# CIS web application firewall (WAF) module
+
+This module enables the web application firewall of the domain.
```

Comment:
It does not enable only. It allows you to disable as well. Hence content needs modification.
Either you say `enable/disable` or `allows to change WAF rule settings of a domain of a CIS instance`.

Reply:
Done